ale/base/label_pds3.py

```diff
@@ -61,26 +61,19 @@ def target_name(self):
         : str
           target name
         """
-
         return self.label['TARGET_NAME']
 
     @property
     def _target_id(self):
         return spice.bodn2c(self.label['TARGET_NAME'])
 
     @property
-    def starting_ephemeris_time(self):
-        if not hasattr(self, '_starting_ephemeris_time'):
-            sclock = self.label['SPACECRAFT_CLOCK_START_COUNT']
-            self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock)
-        return self._starting_ephemeris_time
+    def _starting_ephemeris_time(self):
+        return self.label['SPACECRAFT_CLOCK_START_COUNT']
 
     @property
-    def spacecraft_clock_stop_count(self):
-        sc = self.label.get('SPACECRAFT_CLOCK_STOP_COUNT', None)
-        if sc == 'N/A':
-            sc = None
-        return sc
+    def _ending_ephemeris_time(self):
+        return self.label['SPACECRAFT_CLOCK_STOP_COUNT']
```

Comment:
Same as `_starting_ephemeris_time` this should stay `spacecraft_clock_stop_count` and then the SPICE mix-in can provide `_ending_ephemeris_time`